### PR TITLE
minor code cleanup, std:: prefix, consts, typos

### PR DESCRIPTION
**Pull request by aldenml**

### CODE REVIEW

**Comment by aldenml** [comment]
@arvidn did you have an opportunity to enable "auto cancelation" in travis? (https://blog.travis-ci.com/2017-03-22-introducing-auto-cancellation)

**Comment by arvidn** [comment]
Yes, I definitely attempted to enable it

File: src/utp_stream.cpp [hunk 1]
```diff
@@ -158,7 +158,7 @@ enum
 // into account. if lhs is close to UINT_MAX and rhs
 // is close to 0, lhs is assumed to have wrapped and
 // considered smaller
-TORRENT_EXTRA_EXPORT bool compare_less_wrap(std::uint32_t lhs
+bool compare_less_wrap(std::uint32_t lhs
```
Comment (arvidn) [inline src/utp_stream.cpp]: there's no unit test for this function?

**Comment by arvidn** [comment]
lgtm, apart from ``compare_less_wrap``. I would prefer to add tests for it

**Comment by aldenml** [comment]
Ok, I just only removed the `TORRENT_EXTRA_EXPORT` from the `.cpp`, I don't see any test already, I will add one.

**Comment by arvidn** [comment]
thanks!

**Pull request merged**